docs/projects/adrv9361z7035/index.rst

```diff
@@ -202,7 +202,6 @@ command.
    ============= ===============================================
    ccbob_cmos    ADRV9361Z7035-SOM (CMOS Mode) + ADRV1CRR-BOB
    ccbob_lvds    ADRV9361Z7035-SOM (LVDS Mode) + ADRV1CRR-BOB
-   ccpackrf_lvds ADRV9361Z7035-SOM (LVDS Mode) + ADRV1CRR-PACKRF
```

Comment:
Would be better if you put a * or something, and below the table (but inside the collapsible) to have a note saying that this can be found on the last release branch where it was present.

Reply:
done